src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/ItemsControl.xaml

```diff
@@ -0,0 +1,24 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+    <Style x:Key="DefaultItemsControlStyle" TargetType="{x:Type ItemsControl}">
+        <Setter Property="Border.CornerRadius" Value="4" />
```

Comment:
Shall we remove CornerRadius default to keep these container controls unbiased ?

Reply:
Agreed. I'd keep this to 0 to not break existing styling.

I believe these are 0 in WinUI as well, as these are empty containers with no styling at all